composer.json

```
@@ -48,7 +48,7 @@
         "drupal/config_snapshot": "1.0-rc2",
         "drupal/config_sync": "2.0-beta7",
         "drupal/config_update": "1.7",
-        "drupal/core-recommended": "9.1.7",
+        "drupal/core-recommended": "9.2.x-dev",
```

Alpha1 has been [released](https://www.drupal.org/project/drupal/releases/9.2.0-alpha1).
```suggestion
        "drupal/core-recommended": "9.2.0-alpha1",
```